base/simplebook/ansible/roles/simplebook/tasks/main.yml

```diff
@@ -54,9 +55,30 @@
     - wget
     - xdg-utils
 
-- name: install NodeJS
+- name: Configure redis
+  become_user: root
+  replace:
+    path: "/etc/redis/redis.conf"
+    regexp: 'supervisord no'
+    replace: 'supervisord systemd'
+
+- name: Restart redis
+  become_user: root
+  shell:
+    systemctl restart redis
+
+- name: Prepare NodeJS
+  become_user: root
   shell: |
-    curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - && sudo apt-get install -y nodejs
+    curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
This should be a separate commit -- I removed use of sudo since it made ansible hang / that is not the ansible way of using sudo.

Reply:
Yeah, but if it doesn't end up happening, just add an addendum to the commit message "also slipped into this change, X"